docker/cygnus-ngsi/Dockerfile

```diff
@@ -215,9 +215,6 @@ RUN \
     curl --remote-name --location --insecure --silent --show-error "${FLUME_URL}" && \
     tar zxf apache-flume-${FLUME_VER}-bin.tar.gz && \
     mv apache-flume-${FLUME_VER}-bin ${FLUME_HOME} && \
-    mv ${FLUME_HOME}/lib/httpclient-4.2.1.jar ${FLUME_HOME}/lib/httpclient-4.2.1.jar.old && \
-    mv ${FLUME_HOME}/lib/httpcore-4.2.1.jar ${FLUME_HOME}/lib/httpcore-4.2.1.jar.old && \
-    mv ${FLUME_HOME}/lib/libthrift-0.7.0.jar ${FLUME_HOME}/lib/libthrift-0.7.0.jar.old && \
```

Comment:
PR https://github.com/telefonicaid/fiware-cygnus/pull/1611 touched three Dockerfiles... Maybe the other two needs a similar adaptation?

Reply:
I have checked the cygnus-common/Dockerfile and cygnus-twitter/Dockerfile and it seems they are not includes such mv lines. Thus, NTC.